tools/topology/topology1/CMakeLists.txt

```diff
@@ -316,6 +316,7 @@ set(TPLGS
 
```

Comment:
I think the commit should describe the change a bit more. You can just say add support for new sof-adl-es8336-ssp1-hdmi-ssp02-4ch topology and explain in git commit where this is needed (4ch variant of the SSP codec and HDMI-in capture combo).

Reply:
Updated the commit message a suggested. Please check. Thanks.